src/definitions/elements/input.less

```diff
@@ -477,27 +491,6 @@
   border-right-color: @focusBorderColor !important;
 }
 
-/* Button on Left */
-.ui[class*="left action"].input > input {
-  border-top-left-radius: 0 !important;
-  border-bottom-left-radius: 0 !important;
-  border-left-color: transparent !important;
-}
-.ui[class*="left action"].input > .dropdown,
-.ui[class*="left action"].input > .button,
-.ui[class*="left action"].input > .buttons > .button {
-  border-radius: 0;
-}
-.ui[class*="left action"].input > .dropdown:first-child,
-.ui[class*="left action"].input > .button:first-child,
-.ui[class*="left action"].input > .buttons:first-child > .button {
-  border-radius: @borderRadius 0 0 @borderRadius;
-}
-/* Input Focus */
-.ui[class*="left action"].input > input:focus {
-  border-left-color: @focusBorderColor !important;
-}
```

Comment:
Don't delete this. It's needed. but remove `!important` and increase specificity by one additional `ui` instead, so it becomes

```css
/* Input Focus */
.ui.ui[class*="left action"].input > input:focus {
  border-left-color: @focusBorderColor;
}
```

See https://jsfiddle.net/4L7zvsbc/1/
Try to click into the very bottom input, so it gets focus.

|Your change|Old bevavior|
|-|-|
|![image](https://user-images.githubusercontent.com/18379884/66393684-79a8fd00-e9d3-11e9-91f1-bdc706329a3b.png)|![image](https://user-images.githubusercontent.com/18379884/66393716-8c233680-e9d3-11e9-830b-cb23684a0bd9.png)|